### ISSUE TITLE
AbstractMethodError when trying to pretty print more than one BQ class in a repl session

### ISSUE THREAD

**Issue by andrewsmartin** [spotify/scio#116]
REPL session with two data sets:

```
BigQuery client available as 'bq'
Scio context available as 'sc'

scio> @BigQueryType.fromQuery(
     |     """
     |       | SELECT *
     |       | FROM [bigquery-public-data:samples.shakespeare]
     |     """.stripMargin
     |   )
     |   class Shakespeare
defined class Shakespeare
defined object Shakespeare

scio> Shakespeare.toPrettyString()
res1: String = case class Shakespeare(word: String, word_count: Int, corpus: String, corpus_date: Int)

scio> @BigQueryType.fromQuery(
     |     """
     |       | SELECT *
     |       | FROM [bigquery-public-data:samples.trigrams]
     |     """.stripMargin
     |   )
     |   class Trigrams
[main] INFO com.spotify.scio.bigquery.BigQueryClient - Staging dataset gabocontinuousintegration:scio_bigquery_staging_us already exists
[main] INFO com.spotify.scio.bigquery.BigQueryClient - Creating temporary view gabocontinuousintegration:scio_bigquery_staging_us.scio_query_20160427192123_1139597268
[main] INFO com.spotify.scio.bigquery.BigQueryClient - Deleting temporary view gabocontinuousintegration:scio_bigquery_staging_us.scio_query_20160427192123_1139597268
defined class Trigrams
defined object Trigrams
defined class Cell$
defined class Sample$

scio> Trigrams.toPrettyString()
java.lang.AbstractMethodError: Trigrams$.toPrettyString$default$1()I
  ... 32 elided

scio>
```

**Comment by nevillelyh** [comment]
The original implementation depends on `ClassTag` and messes things up. Re-implemented it in macro land.
```scala
def toPrettyString(indent: Int = 0)(implicit ct: ClassTag[T]): String =
  SchemaUtil.toPrettyString(schema, ct.runtimeClass.getName, indent)
```

**Comment by nevillelyh** [comment]
Fixed in #117
